src/main/scala/com/github/unchama/seichiassist/infrastructure/scalikejdbc/ScalikeJDBCConfiguration.scala

```diff
@@ -17,6 +22,7 @@ object ScalikeJDBCConfiguration {
   )
 
   def initializeConnectionPool(url: String, user: String, password: String): Unit = {
+    Class.forName("org.mariadb.jdbc.Driver")
```

Comment:
一つ前のものと同様です。

```suggestion
    Class.forName("com.github.unchama.seichiassist.relocateddependencies.org.mariadb.jdbc.Driver")
```

nits: これって明示的に読み込む必要はありますか？比較的最近 (JDBC 4.0 以降) のドライバは、基本的にクラスパス上に存在していれば自動的に読み込まれるようです ([参考](https://docs.oracle.com/javadb/10.8.3.0/ref/rrefjdbc4_0summary.html))